src/Azure.Functions.Cli/Actions/LogsActions/GetLogsAction.cs

```diff
@@ -0,0 +1,80 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Azure.Functions.Cli.Actions.DeployActions.Platforms;
+using Azure.Functions.Cli.Common;
+using Azure.Functions.Cli.Helpers;
+using Azure.Functions.Cli.Interfaces;
+using Colors.Net;
+using Fclp;
+using Microsoft.Azure.WebJobs.Script;
+using static Azure.Functions.Cli.Common.OutputTheme;
+using static Colors.Net.StringStaticMethods;
+
+namespace Azure.Functions.Cli.Actions.LocalActions
+{
+    [Action(Name = "logs", HelpText = "Gets logs of Functions running on custom backends")]
+    internal class GetLogsAction : BaseAction
+    {
+        public string Name { get; set; } = string.Empty;
+        public string Platform { get; set; } = string.Empty;
+        public string ConfigPath { get; set; } = string.Empty;
+        private Dictionary<string, Func<string, Task>> logsHandlersMap = new Dictionary<string, Func<string, Task>>();
+        private const string KUBERNETES_DEFAULT_NAMESPACE = "azure-functions";
+
+        private readonly ITemplatesManager _templatesManager;
+
+        public GetLogsAction(ITemplatesManager templatesManager)
+        {
+            _templatesManager = templatesManager;
```

Comment:
You don't really need that, right?

Reply:
@ahmelsayed I certainly dont :)

Committed fix